### PR TITLE
fix: the logic to accept a user provided PEM formatted SSH private key for each git repo domain

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**

### CODE REVIEW

File: common/sshkeys/sshkeys.go [hunk 1]
```diff
@@ -107,65 +107,56 @@ func loadSSHKeysOfCurrentUser() {
 
 	// Ask whether to load private keys or provide own key
 	options := []string{
-		"Load private ssh keys from " + privateKeyDir,
+		fmt.Sprintf("Load the private SSH keys from the directory '%s'" + privateKeyDir),
```
Comment (seshapad) [inline common/sshkeys/sshkeys.go:110]: Where is the privateKeyDir available? Is it provided as input or do we pick up `.ssh` dir on the client machine?

File: common/sshkeys/sshkeys.go [hunk 2]
```diff
@@ -218,61 +202,70 @@ func loadSSHKey(filename string) (string, error) {
...
 func GetSSHKey(domain string) (string, bool) {
 	loadSSHKeysOfCurrentUser()
 	if len(privateKeysToConsider) == 0 {
 		return "", false
 	}
-	if privateKeysToConsider[0] == userProvidedKey {
-		key := qaengine.FetchStringAnswer(common.ConfigRepoPrivKey, "Provide your own PEM-formatted private key:", []string{"Should not be empty"}, "", nil)
+
+	if len(privateKeysToConsider) == 1 && privateKeysToConsider[0] == shoudlAskUserForSSHKey {
+		qaKey := common.JoinQASubKeys(common.ConfigRepoKeysKey, `"`+domain+`"`, "keyData")
```
Comment (seshapad) [inline common/sshkeys/sshkeys.go:226]: Where do we get the domain from?

Comment (HarikrishnanBalagopal) [reply common/sshkeys/sshkeys.go:226]: it's given as input to the function

Comment (seshapad) [reply common/sshkeys/sshkeys.go:226]: I meany where does move2kube get the domain in the first place. Is it an user input?

File: common/sshkeys/sshkeys.go (hunk 1 above)
Comment (seshapad) [reply common/sshkeys/sshkeys.go:110]: Does this function read the `.ssh` directory of the user?

File: common/sshkeys/sshkeys.go (hunk 2 above)
Comment (HarikrishnanBalagopal) [reply common/sshkeys/sshkeys.go:226]: `I meant where does move2kube get the domain in the first place. Is it an user input?` from the `git remote -v` output I think

Comment (seshapad) [reply common/sshkeys/sshkeys.go:226]: Got it. Thanks.

**Review by seshapad — APPROVED** [review]

**Pull request merged**